src/adapters/local.ts

```diff
@@ -42,11 +42,24 @@ export default class LocalAdapter implements IAdapter {
     }
 
     async rename(oldPath: string, newPath: string): Promise<boolean> {
-        throw new Error('rename NYI');
+        await fs.renameAsync(this.fullPath(oldPath), this.fullPath(newPath));
+
+        return true;
```

Comment:
so yeah, we need to go through and decide what we want stuff to return. Just `true` or `Metadata`?

And so you know, the return type is `Promise<boolean>`, where the stuff in the `<>` is what the promise resolves to. So we're considering changing it to `Promise<IMetadata>`. I don't like the `ISomething` pattern for interfaces, but it's apparently pretty standard in TypeScript, and I want people familiar with TypeScript to be able to use these interfaces.